src/components/NearbyLocationsList.js

```diff
@@ -29,21 +28,22 @@ type State = {|
 // todo add onItemPress
 // add onPourIconPress
 @withNavigation
-@inject('nearbyLocationsStore')
 @observer
 class TapsList extends InjectedComponent<InjectedProps, {}, State> {
+  _nearbyLocationsStore: NearbyLocationsStore = new NearbyLocationsStore();
```

Comment:
Shouldn't this store be a singleton?

Reply:
this store will be removed at all when you added support for nearbyLocations in odata